packages/ui/src/styles.css

```diff
@@ -100,6 +100,42 @@
     --diff-empty-content--: var(--background);
     --diff-hunk-content-color--: hsl(0, 0%, 100%);
     /* New colors design variables */
+
+    --canary-harness-nav-panel-bg: 212, 72%, 10%;
```

Comment:
ideally these colors should not be in the `ui` package. they should be co-located with the corresponding apps. but not a blocker for now